offset/offset.go

```diff
@@ -0,0 +1,88 @@
+// Package offset provides functions for validating request offsets.
+package offset
```

Comment:
Not required - chs-streaming-api-cache should be responsible for reporting whether or not a given offset number is valid and the frontend should be responsible for handling the error that it returns (if any).